osa/provenance/utils.py

```diff
@@ -62,69 +70,92 @@ def parse_variables(class_instance):
             calibration_path / timecalibration_filename
         )
         class_instance.PointingFile = class_instance.args[3]
-        class_instance.ObservationRun = class_instance.args[5].split(".")[0]
-        class_instance.ObservationSubRun = class_instance.args[5].split(".")[0]
+        class_instance.RunSummaryFile = os.path.basename(class_instance.args[4])
```

Comment:
I think here we could just set `str(class_instance.args[4])` in order to show the abs path